src/connector/src/sink/mysql.rs

```diff
@@ -125,8 +149,57 @@ impl TryFrom<Datum> for MySQLValue {
     }
 }
 
+#[allow(clippy::upper_case_acronyms)]
+#[derive(strum_macros::Display)]
+enum MySQLDataType {
+    BOOL,
+    SMALLINT,
+    INT,
+    BIGINT,
+    FLOAT,
+    DOUBLE,
+    // TODO(nanderstabel): find solution for varchar length.
+    #[strum(serialize = "VARCHAR(255)")]
+    VARCHAR,
+}
+
+impl From<&DataType> for MySQLDataType {
+    fn from(data_type: &DataType) -> MySQLDataType {
+        match data_type {
+            DataType::Boolean => MySQLDataType::BOOL,
+            DataType::Int16 => MySQLDataType::SMALLINT,
+            DataType::Int32 => MySQLDataType::INT,
+            DataType::Int64 => MySQLDataType::BIGINT,
+            DataType::Float32 => MySQLDataType::FLOAT,
+            DataType::Float64 => MySQLDataType::DOUBLE,
+            DataType::Varchar => MySQLDataType::VARCHAR,
+            _ => unimplemented!(),
+        }
+    }
+}
+
 #[async_trait]
 impl Sink for MySQLSink {
+    async fn prepare(&mut self, schema: &Schema) -> Result<()> {
```

Comment:
I really doubt whether we should do that, cc @neverchanje .

Reply:
Perhaps adding this prepare function is not the best way. Another solution is to move the "prepare" implementation to `MySQLSink::new`. This might make more sense and won't require other Sinks to implement a prepare function. The only change this will require is that `schema` needs to be added to `fn build_sink` and `SinkImpl::new` signature. Wdyt @tabVersion ?